.travis/install_ubuntu_dependencies.sh

```diff
@@ -16,9 +16,10 @@
 set -ex
 
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+sudo add-apt-repository ppa:deadsnakes/ppa -y
```

Comment:
Why do we need additional ppa? Is there a hard dependency on python3.6 specifically?